app-editors/bvi/bvi-1.4.0.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="display-oriented editor for binary files, based on the vi texteditor"
```

Comment:
Please start with a capital letter here.

Reply:
This was copied from the previous version. Fixed in 5506daf89481.